checks/raw/binary_artifact.go

```diff
@@ -30,7 +30,7 @@ import (
 
 // BinaryArtifacts retrieves the raw data for the Binary-Artifacts check.
 func BinaryArtifacts(c clients.RepoClient) (checker.BinaryArtifactData, error) {
-	var files []checker.File
+	files := []checker.File{}
```

Comment:
If you want to create an empty slice prefer: `make([]checker.File, 0)` to make the intent explicit.

Reply:
thanks.